invenio_base/wsgi.py

```diff
@@ -53,11 +55,51 @@ def wsgi_proxyfix(factory=None):
        otherwise you application is susceptible to malicious attacks.
 
     .. versionadded:: 1.0.0
+
+    .. note::
+
+       You must set ``APP_PROXY_CONFIG`` with the correct values if your
```

Comment:
I am not sure I understand fully the explanation :)

What about something like (please fix my english, it is just to give the idea)
```
`APP_PROXY_CONFIG` lets you specify the number of proxies that an HTTP request is traversing before reaching the Invenio app. Invenio will try to read the request headers to guess for example host, protocol or scheme. To avoid any possible malicious attack that is manipulating headers and trust instead your proxies headers, specify this configuration as the following:
```